yarn.lock

```diff
@@ -25,6 +25,13 @@
   resolved "https://registry.yarnpkg.com/@ardatan/aggregate-error/-/aggregate-error-0.0.1.tgz#1403ac5de10d8ca689fc1f65844c27179ae1d44f"
```

Comment:
More green than red ;(

Reply:
`graphql-config` major bump (that's dep of `eslint-plugin-graphql`) added quite a bit of `graphql-tools/*` packages to the mix :(